pom.xml

```diff
@@ -95,7 +95,7 @@
 
         <!-- Test Dependency versions -->
         <testng.version>6.10</testng.version>
-        <mockito-all.version>1.8.5</mockito-all.version>
+        <mockito-all.version>4.11.0</mockito-all.version>
```

Comment:
We are not using mockito-all, better rename this to mockito.version

Reply:
sure @stoty let me update

Reply:
done